setup/setuputils.class.inc.php

```diff
@@ -897,9 +897,13 @@ public static function CheckDiskSpace($sDir)
 		return $f;
 	}
 
+    /**
+     * @link https://en.wiktionary.org/wiki/petabyte petabyte PB
+     * @link https://en.wiktionary.org/wiki/exabyte#English exabyte EB
+     */
 	public static function HumanReadableSize($fBytes)
 	{
-		$aSizes = array('bytes', 'Kb', 'Mb', 'Gb', 'Tb', 'Pb', 'Hb');
+        $aSizes = array('bytes', 'Kb', 'Mb', 'Gb', 'Tb', 'Pb', 'Eb');
```

Comment:
Actually, the first one says bytes and all the others are about bits?
```suggestion
        $aSizes = array('bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB');
```

See https://en.wiktionary.org/wiki/EB#Symbol vs https://en.wiktionary.org/wiki/Eb#Symbol

Reply:
Good catch!

Reply:
You're perfectly right ! I was wondering also why we had "b" and not "B" ?
I've just checked, values processed are from `disk_free_space()` of `filesize()`, so they are indeed in bytes.
I'll fixed that, thanks

Reply:
Fixed in 684fa2d4